gensim/models/ensemblelda.py

```diff
@@ -0,0 +1,1298 @@
+    >>> elda = EnsembleLda(corpus=common_corpus, id2word=common_dictionary, num_topics=10, num_models=4)
+
+Save a model to disk, or reload a pre-trained model
+
+.. sourcecode:: pycon
+
+    >>> from gensim.test.utils import datapath
+    >>>
+    >>> # Save model to disk.
+    >>> temp_file = datapath("model")
+    >>> elda.save(temp_file)
+    >>>
+    >>> # Load a potentially pretrained model from disk.
+    >>> elda = EnsembleLda.load(temp_file)
+
+Query, the model using new, unseen documents
+
+.. sourcecode:: pycon
+
+    >>> # Create a new corpus, made of previously unseen documents.
+    >>> other_texts = [
+    ...     ['computer', 'time', 'graph'],
+    ...     ['survey', 'response', 'eps'],
+    ...     ['human', 'system', 'computer']
+    ... ]
+    >>> other_corpus = [common_dictionary.doc2bow(text) for text in other_texts]
+    >>>
+    >>> unseen_doc = other_corpus[0]
+    >>> vector = elda[unseen_doc]  # get topic probability distribution for a document
+
+Increase the ensemble size by adding a new model. Make sure it uses the same dictionary
+
+.. sourcecode:: pycon
+
+    >>> from gensim.models import LdaModel
+    >>> elda.add_model(LdaModel(common_corpus, id2word=common_dictionary, num_topics=10))
+    >>> elda.recluster()
+    >>> vector = elda[unseen_doc]
+
+To optimize the ensemble for your specific case, the children can be clustered again using
+different hyperparameters
+
+.. sourcecode:: pycon
+
+    >>> elda.recluster(eps=0.2)
+
+References
+----------
+.. [1] REHUREK, Radim and Sojka, PETR, 2010, Software framework for topic modelling with large corpora. In : THE LREC
+       2010 WORKSHOP ON NEW CHALLENGES FOR NLP FRAMEWORKS [online]. Msida : University of Malta. 2010. p. 45-50.
+       Available from: http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.695.4595
+
+.. [2] BRIGL, Tobias, 2019, Extracting Reliable Topics using Ensemble Latent Dirichlet Allocation [Bachelor Thesis].
+       Ingolstadt: Technische Hochschule. Available from: https://www.hip70890b.de/machine_learning/ensemble_LDA/
+
+Citation
+--------
+At the moment, there is no paper associated to ensemble LDA but we are working on publicly releasing Tobi Brigl's
+bachelor thesis on the topic.  In the meantime, please include a mention of us (Brigl, T.; Salles, A.; Loosley, A) and
+a link to this file (https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/ensemblelda.py) if you use
+this work in a published or open-source project.
+
+Other Notes
+-----------
+.. The adjectives stable and reliable (topics) are used somewhat interchangeably throughout the doc strings and
+comments.
+
+"""
+
+import logging
+import os
+from multiprocessing import Process, Pipe, ProcessError
+import numpy as np
+from scipy.spatial.distance import cosine
+from gensim import utils
+from gensim.models import ldamodel, ldamulticore, basemodel
+
+logger = logging.getLogger(__name__)
+
+
+class EnsembleLda():
```

Comment:
Inherit from `object` (for Python 2).

Reply:
But we don't care about Python 2 anymore, right?

Reply:
In that case, drop the `()`.